packages/server-api/src/index.ts

```diff
@@ -170,4 +172,10 @@ export interface ServerAPI extends PluginServerApp {
     ) => void
   }) => void
   getSerialPorts: () => Promise<Ports>
+  getCourse: () => CourseInfo
+  clearDestination: () => void
```

Comment:
In my experience "everything should be async / return a Promise" so I think these 2 should also return promises.

Reply:
The MVP is that stream processing plugins can reflect the course information received from NMEA data in SignalK.
So these methods would be the minimum.
The others could be added as required, I'm guessing updating plugins to use the interface will identify the need ,or not.

Reply:
I will make the 2 functions Async.